deploy/kubernetes/setup.sh

```diff
@@ -162,6 +182,9 @@ kubectl -n sumologic create secret generic sumologic \
   --from-literal=endpoint-events=$ENDPOINT_EVENTS
 
 echo "Applying deployment 'fluentd'..."
-kubectl apply -f https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
+sed 's/\$NAMESPACE'"/$NAMESPACE/g" | \
+tee sumologic-collection.yaml | \
+kubectl -n $NAMESPACE apply -f sumologic-collection.yaml
```

Comment:
I like the name `sumologic-collection`, should we consider renaming `fluentd-sumologic.yaml.tmpl` to `sumologic-collection.yaml.tmpl` as well? cc @frankreno @abhi-sumo

Reply:
Or the other way around? I should leave the generated file as `fluentd-sumologic.yaml`?

Reply:
Actually, on second thought any customer that deploys manually will need to deploy this yaml for fluentd, then deploy fluentbit and prometheus separately. So I think it makes more sense to leave the name as `fluentd-sumologic`, sorry @fernandesnikhil do you mind changing it back?

Reply:
Done.